src/main/kotlin/no/nav/familie/oppdrag/iverksetting/OppdragSenderMock.kt

```diff
@@ -1,37 +0,0 @@
-package no.nav.familie.oppdrag.iverksetting
```

Comment:
Må du kanskje beholde denne fila? Hvis ikke: Kan den tilsvarende mocken for AvstemmingSender også fjernes?

Reply:
.... ja riktig .. mocken er fjernet ettersom den ikke er i bruk som følge av at service mockes ut ... Men Avstemningsservicen mockes ikke ut derfor har jeg beholdt sendermocken for denne .. Men syns du vi skal være konsistente og mcoke ut avsteminingservice også? ...